Main.elm

```diff
@@ -1,57 +1,475 @@
+module Main exposing (main)
+
 import Html exposing (..)
 import Html.Attributes exposing (..)
+import Bootstrap.Card as Card
+import Bootstrap.Grid as Grid
 import Html.Events exposing (onInput)
+import Html exposing (Html, button, div, text)
+import Html.Events exposing (onClick)
+import Bootstrap.Grid.Col as Col
+
 
+-- MAIN
 
+
+main : Program Never Model Msg
 main =
-  Html.beginnerProgram { model = model, view = view, update = update }
+    Html.program
+        { view = view
+        , update = update
+        , subscriptions = always Sub.none
+        , init = init
+        }
 
 
--- MODEL
 
-type alias Model =
-  { name : String
-  , city : String
-  , country : String
-  }
+-- INITIALISATION
 
 
-model : Model
-model =
-  Model "" "" ""
+type alias Model =
```

Comment:
Basically, what is important is to have explicit variables names, in english to avoid mixing French (from variables) and english (from the language vocabulary)